Source/Core/Core/HW/EXI/BBA/BuiltIn.h

```diff
@@ -34,6 +34,23 @@ struct TcpBuffer
   std::vector<u8> data;
 };
 
+// Socket helper class to ensure network interface consistency
```

Comment:
What does this comment mean.  I see a tcp/udp socket with different interface functions?

Reply:
If the socket (TCP/UDP doesn't matter) isn't bound, the system will pick the interface to use automatically. This can be an issue if you have multiple interfaces/IP addresses that can reach the socket's remote peer. In other words, your source IP address might change when talking to the same peer.

Without this fix, Windows will randomly pick my IP address or my VMware network interface IP (which can be observed in BBA PCAP dump), on my machine.

That's a similar issue as the one mentioned in https://github.com/dolphin-emu/dolphin/pull/9735.